docs/cn/instructions/eventmesh-runtime-protocol.md

```diff
@@ -258,3 +258,123 @@ public class LiteMessage {
 | 场景               | Server向Client发送消息请求码 | Client回复Server消息响应码 | 说明                   |
 | ------------------ | ---------------------------- | -------------------------- | ---------------------- |
 | 客户端接收异步事件 | HTTP_PUSH_CLIENT_ASYNC(105)  | retCode                    | retCode值为0时代表成功 |
+
+
+## gRPC 协议文档
+
+#### 1. protobuf
+
+在 `eventmesh-protocol-gprc` 模块有 Eventmesh gRPC 客户端的 protobuf 文件. the protobuf 文件路径是 `/src/main/proto/eventmesh-client.proto`.
+
+用gradle build 生成 gRPC 代码在 `/build/generated/source/proto/main`. 生成代码用于 `eventmesh-sdk-java` 模块.
+
+#### 2. gRPC 数据模型
+
+- 消息
+
+以下消息数据模型用于 `publish()`, `requestReply()` 和 `broadcast()` APIs.
+
+```
+message RequestHeader {
+    string env = 1;
+    string region = 2;
+    string idc = 3;
```

Comment:
region, namespace, idc, rack and so on. these tenant or geographic info should be carefully designed in our v2 version.

Reply:
BTW, any guys want to add comment for every field in protocol?

Reply:
This is V1 version of protobuf and we are experimenting this grpc transport feature.

Please  review and comment on this set of header attributes and improve it in V2. @qqeasonchen @ruanwenjun 

Thanks.